README.md

```diff
@@ -2,6 +2,12 @@
 
 ![CI](https://github.com/hCaptcha/hcaptcha-android-sdk/workflows/Android%20SDK%20CI/badge.svg)
 [![Release](https://jitpack.io/v/hCaptcha/hcaptcha-android-sdk.svg)](https://jitpack.io/#hCaptcha/hcaptcha-android-sdk)
+[![Minimal Android OS](https://img.shields.io/badge/Android%20OS%20-%3E=4.1-blue.svg)](https://developer.android.com/about/dashboards)
+
+| Platform   | Requirements                                 |
```

Comment:
This section can go under a `Requirements` after the `Installation`.

![image](https://user-images.githubusercontent.com/11096226/181011625-76de2617-811b-4dc4-8fa9-752dd6b0d569.png)

Reply:
Done